modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-using-custom-scc-to-improve-prod-security_{context}"]
+= Using a custom SCC to improve pod security
+
+To improve the security of your builds and runs, you can use custom security context constraints (SCCs).
+
+.Procedure
+
+* Edit the `pipelines-scc` and modify the `runAsUser` and `seLinuxContext`:
```

Comment:
IMHO it's safer to create a new SCC. This one is managed by operator so it could be removed or changed after updating OpenShift Pipelines.